README.md

```
@@ -1,34 +1,36 @@
-  <a href="https://www.gnu.org/licenses/agpl-2.0">
-    <img src="https://img.shields.io/badge/License-AGPL%20v2-blue.svg" alt="License: AGPL v2" />
-  </a>
-  <a href="SECURITY.md">
-    <img src="https://img.shields.io/badge/Security-Policy-blue.svg" alt="Security Policy" />
-  </a>
-</p>
-<br>
-<hr>
+<div align="center" style="border-bottom:1px solid #444;width:100%;">
+
+  <img src="./docs/assets/images/nanodns.webp" style="text-align:center;" width="200"/>
```

Markdownlint

Images should have alternate text (alt text)

(MD045, no-alt-text)